shell-completion/bash/systemd-cryptenroll

```diff
@@ -50,7 +50,7 @@ _systemd_cryptenroll() {
     local cur=${COMP_WORDS[COMP_CWORD]} prev=${COMP_WORDS[COMP_CWORD-1]} words cword
     local -A OPTS=(
         [STANDALONE]='-h --help --version
-                     --password --recovery-key'
+                     --password --recovery-key --list-devices'
```

Comment:
i guess __get_block_devices above should actually be changed to use `systemd-cryptenroll --list-devices` to generate the list of suitable devices.

Reply:
Indeed, very helpful.